tests/Integration/Behaviour/Features/Scenario/Search/add_alias.feature

```diff
@@ -0,0 +1,101 @@
+      | alias   | large |
+      | search  | big   |
+    Then following aliases should exist:
+      | id reference | alias  | search |
+      | alias1       | bloose | blouse |
+      | alias2       | blues  | blouse |
+      | alias3       | large  | big    |
+
+  Scenario: I add a multiple aliases with basic information
+    Given following aliases should exist:
+      | id reference | alias  | search |
+      | alias1       | bloose | blouse |
+      | alias2       | blues  | blouse |
+      | alias3       | large  | big    |
+    When I add alias with following information:
+      | alias   | bliu,blu |
+      | search  | blue     |
+    Then following aliases should exist:
+      | id reference | alias  | search |
+      | alias1       | bloose | blouse |
+      | alias2       | blues  | blouse |
+      | alias3       | large  | big    |
+      | alias5       | bliu   | blue   |
+      | alias6       | blu    | blue   |
+
+  Scenario: I add a multiple aliases with duplicate search field
+    Given following aliases should exist:
+      | id reference | alias  | search |
+      | alias1       | bloose | blouse |
+      | alias2       | blues  | blouse |
+      | alias3       | large  | big    |
+      | alias5       | bliu   | blue   |
+      | alias6       | blu    | blue   |
+    When I add alias with following information:
+      | alias   | bliu,sapphire |
+      | search  | blue          |
+    Then following aliases should exist:
+      | id reference | alias    | search |
+      | alias1       | bloose   | blouse |
+      | alias2       | blues    | blouse |
+      | alias3       | large    | big    |
+      | alias5       | bliu     | blue   |
+      | alias6       | blu      | blue   |
+      | alias7       | sapphire | blue   |
+
+    When I add alias with following information:
+      | alias   | blah |
+      | search  | blue |
+    Then following aliases should exist:
+      | id reference | alias    | search |
+      | alias1       | bloose   | blouse |
+      | alias2       | blues    | blouse |
+      | alias3       | large    | big    |
+      | alias5       | bliu     | blue   |
+      | alias6       | blu      | blue   |
+      | alias7       | sapphire | blue   |
+      | alias8       | blah     | blue   |
+
+  Scenario: I add alias with empty alias field
+    When I add alias with following information:
+      | alias   |      |
+      | search  | blue |
+    Then following aliases should exist:
+      | id reference | alias    | search |
+      | alias1       | bloose   | blouse |
+      | alias2       | blues    | blouse |
+      | alias3       | large    | big    |
+      | alias5       | bliu     | blue   |
+      | alias6       | blu      | blue   |
+      | alias7       | sapphire | blue   |
+      | alias8       | blah     | blue   |
+
+  Scenario: I add alias with empty search field
+    When I add alias with following information:
+      | alias   | blu |
+      | search  |     |
```

Comment:
here aswell, I think now the error comes from legacy validator, but it can fail faster on commad constructor level and throw invalid argument exception when search term is empty.